src/Orchard.Web/Modules/Orchard.Workflows/ResourceManifest.cs

```diff
@@ -8,23 +9,36 @@
 namespace Orchard.Workflows {
     public class ResourceManifest : IResourceManifestProvider {
         private readonly Work<IActivitiesManager> _activitiesManager;
+        private readonly IHostEnvironment _hostEnvironment;
 
-        public ResourceManifest(Work<IActivitiesManager> activitiesManager) {
+        public ResourceManifest(Work<IActivitiesManager> activitiesManager, IHostEnvironment hostEnvironment) {
             _activitiesManager = activitiesManager;
+            _hostEnvironment = hostEnvironment;
         }
 
         public void BuildManifests(ResourceManifestBuilder builder) {
             var manifest = builder.Add();
 
             manifest.DefineStyle("WorkflowsAdmin").SetUrl("orchard-workflows-admin.css").SetDependencies("~/Themes/TheAdmin/Styles/Site.css");
 
-            var activityNames = _activitiesManager.Value.GetActivities().Select(x => "WorkflowsActivity-" + x.Name).ToArray();
+            var resourceNames = new List<string>();
 
-            foreach (var activityName in activityNames) {
-                manifest.DefineStyle(activityName).SetUrl(activityName.HtmlClassify()).SetDependencies("WorkflowsAdmin");
+            foreach (var activity in _activitiesManager.Value.GetActivities()) {
+                var assemblyName = activity.GetType().Assembly.GetName().Name;
+                var basePath = VirtualPathUtility.AppendTrailingSlash("~/Modules/" + assemblyName + "/styles/");
```

Comment:
It's technically possible for a Theme to define Workflow activities, but we won't find them this way, because "Modules" is wired in.